crates/runtime/src/logic.rs

```diff
@@ -142,6 +142,17 @@ impl<'a> VMLogic<'a> {
         }
         .build()
     }
+
+    /// Modifies a value through the blockchain bridge
+    ///
+    /// # Parameters
+    ///
+    /// * `value` - The value to be modified on the blockchain.
+    ///
+    pub fn modify_value(&mut self, _value: Vec<u8>) -> VMLogicResult<()> {
+        // TODO: Implement actual blockchain call
+        Ok(())
+    }
```

Comment:
Here we should parse the [proxy contract actions](https://github.com/calimero-network/core/pull/854/files#diff-d7aee4a31d8be649fe544572f6a08467e3bbfa6780d448eebb3047346a9d1e9cR68), not just `modify_value`

Reply:
this should be submitProposal(actions)